app/javascript/packs/application.js

```diff
@@ -24,21 +24,15 @@ require("channels")
 
 // External imports
 import "bootstrap";
-import "../plugins/flatpickr";
+import flatpickr from "flatpickr";
+
 import { initMapbox } from '../plugins/init_mapbox';
 import { initAutocomplete } from '../plugins/init_autocomplete';
 
 document.addEventListener('turbolinks:load', () => {
   initMapbox();
   initAutocomplete();
+  flatpickr(".datepicker", {
```

Comment:
cf lecture of this morning. Flatpick should be activate when turbolinks is loaded meaning when your change page